tests/ui/coherence/coherence-overlap-negative-cycles.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0119]: conflicting implementations of trait `MyTrait`
+  --> $DIR/coherence-overlap-negative-cycles.rs:14:1
+   |
+LL | impl MyTrait for Foo where Foo: MyTrait {}
+   | ^^^^^^^^^^^^^^^^^^^^
+   | |
+   | first implementation here
+   | conflicting implementation
```

Comment:
Ideally we'd find the impl that satisfies `!MyTrait` and highlight it -- can we do that with `SelectionContext::select`? This could be a follow-up I guess..

Reply:
Yes, diagnostics are not really handled in the PR yet. Is one of the things that are still not complete.